modules/core/client/components/LanguageSwitch.component.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import i18n from './i18n';
+
+export default function LanguageSwitch() {
+  const changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
```

Comment:
Let's avoid shortening things when possible to make code easier to read? `code` or `language` would read nicely :-)